apps/examples/testcase/ta_tc/security/utc/utc_crypto.c

```diff
@@ -196,7 +232,7 @@ static void utc_crypto_aes_encryption_output_n(void)
 {
 	security_data enc = {NULL, 0};
 
-	unsigned char iv[] = "tamp_iv_value";
+	unsigned char iv[] = "temp_iv_value";
```

Comment:
This looks not a part of what you explain in the commit description.
If it's just typo fix, let's split a commit.